pkg/server/apiextensions.go

```diff
@@ -217,18 +224,25 @@ func (c *apiBindingAwareCRDLister) ListWithContext(ctx context.Context, selector
 			}
 
 			// Priority 2: Add APIBinding CRDs. These take priority over those from the local workspace.
+
+			// Add the APIExport identity hash as an annotation to the CRD so the RESTOptionsGetter can assign
+			// the correct etcd resource prefix.
+			crd = crd.DeepCopy()
```

Comment:
if you don't mind, would rather just do 
```
shallowCopy := *crd
shallowCopy.Annotations = <some-map-deepcopy>
....
```
CRDs are giant. This is a pretty heavy operation.

Reply:
Great catch, thanks. Fixing

Reply:
still open